components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/cache/AuthenticationContextCache.java

```diff
@@ -86,13 +86,18 @@ public void addToCache(AuthenticationContextCacheKey key, AuthenticationContextC
 
             if (entry.getContext() != null && entry.getContext().getProperties() != null) {
                 Iterator it = entry.getContext().getProperties().entrySet().iterator();
-                while (it.hasNext())
-                {
-                    Map.Entry<String, Object> item =  (Map.Entry<String, Object>) it.next();
+                while (it.hasNext()) {
+                    Map.Entry<String, Object> item = (Map.Entry<String, Object>) it.next();
                     if (!(item.getValue() instanceof Serializable)) {
                         it.remove();
                     }
                 }
+                if (log.isDebugEnabled()) {
+                    String message = "[ Context Id : " + key.getContextId() +
+                            ", Cache type : " + AUTHENTICATION_CONTEXT_CACHE_NAME +
```

Comment:
Let's use string builder here since more than 4 strings are appended.

Reply:
Unless its inside a loop i guess its fine to use string concatenation. Because internally compiler changes it to use string builder when we concat using +
https://dzone.com/articles/string-concatenation-performacne-improvement-in-ja